File: front/src/components/authentication/dtos/add-user-existing-company-dto.ts
import { AddressDto } from "shared/components/address/dtos/address-dto";
import { ContactDto } from "shared/components/contact/dtos/contact-dto";

export class AddUserExistingCompanyDto {
  companyAuthId!: number;
  email!: string;
  userName!: string;
  password!: string;
  address!: AddressDto;
  contact!: ContactDto;
}

File: back/Application/Services/Operations/Auth/Dtos/AddUserExistingCompanyDto.cs
using Application.Services.Shared.Dtos;

namespace Application.Services.Operations.Auth.Dtos;

public class AddUserExistingCompanyDto
{
    public int companyAuthId { get; set; }
    public required string Email { get; set; }
    public required string UserName { get; set; }
    public required string Password { get; set; }
    public AddressDto? Address { get; set; }
    public ContactDto? Contact { get; set; }

}